models/public/colorization-v2-norebal/colorization-v2-norebal.md

```diff
@@ -0,0 +1,111 @@
+# colorization-v2-norebal
+
+## Use Case and High-Level Description
+
+The `colorization-v2-norebal` model is one of the [colorization](https://arxiv.org/pdf/1603.08511)
+group of models designed to perform image colorization. For details
+about this family of models, check out the [repository](https://github.com/richzhang/colorization).
+
+The gray-scale or BGR image should be preprocessed first:
+1) divide image's values by 255
+2) convert the image to LAB
```

Comment:
Isn't the former just a part of the latter?

Reply:
@IRDonch corrected